### PR TITLE
docathon: update README with readthedocs

### PR DESCRIPTION
**Pull request by robrap**
Updates README with references to readthedocs devstack documentation.

Note: The updates here are reasonably minimal. Far more could be done to refactor and slim down the README by taking advantage of the docs, but that is being left for later.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -52,22 +57,25 @@ Where to Find Help
...
+You can also browse all the documentation in `Read the Docs`_.
 
-You can also browse other specific docs in the `docs directory`_.
+.. _most common workflow: https://edx.readthedocs.io/projects/open-edx-devstack/en/latest/workflow.html
+.. _helpful troubleshooting tips: https://edx.readthedocs.io/projects/open-edx-devstack/en/latest/troubleshoot_general_tips.html
+.. _Frequently Asked Questions: https://edx.readthedocs.io/projects/open-edx-devstack/en/latest/devstack_faq.html
+.. _testing and debugging your code in devstack:
+.. _testing_and_debugging.rst: https://edx.readthedocs.io/projects/open-edx-devstack/en/latest/testing_and_debugging.html
 
-.. _workflow.rst: docs/workflow.rst
```
Comment (regisb) [inline README.rst:62]: This link is required at line 280, which causes an error while building the docs:

```
README.rst:280: D000 Unknown target name: "workflow.rst".
docs/readme.rst:280: D000 Unknown target name: "workflow.rst".
```

Comment (robrap) [reply README.rst:62]: Thanks. Sorry - I squashed the fix.  I updated `most common workflow` to `most common development workflow`, and reused it in the second location that had this target.

File: README.rst [hunk 2]
```diff
@@ -52,22 +57,25 @@ Where to Find Help
...
-.. _troubleshoot_general_tips.rst: docs/troubleshoot_general_tips.rst
-.. _devstack_faq.rst: docs/devstack_faq.rst
-.. _testing_and_debugging.rst: docs/testing_and_debugging.rst
-.. _docs directory: docs/
+.. _Read the Docs: https://edx.readthedocs.io/projects/open-edx-devstack/en/latest/
 
 Notices
 -------
 
+**NOTE:** LMS is now using MySql 5.7 by default. You have to run ``make dev.pull.lms`` and ``make dev.provision.lms`` (more details in `Getting Started`_) to fetch latest images and reprovision local copies of databases in order for an existing devstack setup to keep working.
```
Comment (regisb) [inline README.rst]: I believe this line is incorrectly duplicated, right?

Comment (robrap) [reply README.rst]: Thanks. Squashed again because the fix was obvious and the build takes so long. :)

**Review by regisb — APPROVED** [review]

**Pull request merged**